packages/react-components/stories/item-unknown.stories.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react';
+import { ItemUnknown, SimpleInfo } from '../lib';
+
+export default {
+  title: 'Item Unknown',
+  component: ItemUnknown,
+} as Meta;
+
+export const LiftsPanel: Story = (args) => {
```

Comment:
getting an error here 
![image](https://user-images.githubusercontent.com/11761240/104225279-f28fd380-5424-11eb-9a7a-560925cd3429.png)

```
Cannot read property 'displayName' of undefined
TypeError: Cannot read property 'displayName' of undefined
    at displayName (http://localhost:6006/vendors~main.169c47d3206e22fd5981.bundle.js:60084:22)
    at parseReactElement (http://localhost:6006/vendors~main.169c47d3206e22fd5981.bundle.js:228970:21)
    at http://localhost:6006/vendors~main.169c47d3206e22fd5981.bundle.js:228986:12
    at Array.map (<anonymous>)
    at parseReactElement (http://localhost:6006/vendors~main.169c47d3206e22fd5981.bundle.js:228985:106)
    at reactElementToJsxString (http://localhost:6006/vendors~main.169c47d3206e22fd5981.bundle.js:229449:21)
    at http://localhost:6006/vendors~main.169c47d3206e22fd5981.bundle.js:60097:76
    at mapSingleChildIntoContext (http://localhost:6006/vendors~main.169c47d3206e22fd5981.bundle.js:236638:26)
```
Just checking, Is `LiftsPanel` the name you want?

Reply:
Changed the story name.

I can't seem to replicate this error on my environment. Are there any more detailed error messages or perhaps you can update your environment just to rule out that the error is not from your side?